model/godleyTableWindow.cc

```diff
@@ -903,12 +914,12 @@ namespace minsky
 
   void GodleyTableWindow::checkCell00()
   {
-    if (selectedCol==0 && selectedRow==0)
-      // (0,0) cell not editable
+    if ((selectedCol==0 && selectedRow==0) || (selectedCol==0 && selectedRow==1))
```

Comment:
Simplify to selectedCol==0 && (selectedRow==0 || selectedRow==1), or even
selectedCol==0 && selectedRow>1

Reply:
I'm using your second suggestion, thanks :)